connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -1694,33 +1702,41 @@ ble_error_t Gap::createAdvertisingSet(
 #if BLE_FEATURE_EXTENDED_ADVERTISING
 ble_error_t Gap::destroyAdvertisingSet(advertising_handle_t handle)
 {
+    tr_info("destroying advertising set %d", handle);
```

Comment:
I think it's better to trace the destruction when it's successful only

Reply:
If a user post a log for a bug, I'd prefer to have a log that allows me to understand what the application was doing. 
Info on operations started is very important for us. It also provide additional context for error traces.

Reply:
there are traces for all return values so all attempts will be logged